README.md

```diff
@@ -1 +1,87 @@
-# microverse_portfolio
\ No newline at end of file
+Portfolio-Project
+
+📗Table of Contents
+
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+💻 Getting Started
+Prerequisites
+Setup
+Usage
+👥 Authors
+👥 Future Features
+🤝 Contributing
```

Comment:
- [ ]  Well done adding a README to make a professional readme file; kindly customize the sections, and fill the sections as you can based on this milestone project. And the content is not well arranged as the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md), so [use markdown ](https://www.markdownguide.org/basic-syntax/) to arrange those issues.